book.rb

```diff
@@ -0,0 +1,13 @@
+class Book
+  attr_accessor :title, :author
+
```

Comment:
Let's make `rentals` also available in `attr_accessor`

Reply:
Done